<!DOCTYPE HTML>
<!--
	Escape Velocity by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Verbruggen lab - Publications</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<style>
			tr:nth-child(odd) {background-color: #E8E8E8;}
			td {padding: 10px;}
			table {font-family: "Verdana", Helvetica, sans-serif; color: #444444; font-size: 75%; line-height: 150%;}
			h2 {background:#444444; color: #FFFFFF; padding:5px 10px;}
		</style>
	</head>
	<body class="no-sidebar is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<section id="header" class="wrapper">

					<!-- Logo -->
						<div id="logo">
							<h1><a href="index.html">Verbruggen laboratory</a></h1>
							<p>Algal evolution and Coral holobiont biology</p>
						</div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="index.html">Home</a></li>
								<li><a href="people.html">People</a></li>
								<li><a href="research.html">Research</a></li>
								<li><a href="publications.html">Publications</a></li>
								<li><a href="resources.html">Resources</a></li>
							</ul>
						</nav>

				</section>

			<!-- Main -->
				<div id="main" class="wrapper style2">
					<div class="title">Publications</div>
					<div class="container">

						<!-- Content -->
							<div id="content">
								<article class="box post">

<h2>Submitted &ndash; accepted &ndash; in press</h2>
<table>
	<tr><td>
		Bjornson S., Bringloe T.T., Uthanumallian K., De Paoli-Iseppi R., Clark M.B., Cremen C.M.C., Hossen R., McGinley O., Jalali T., West J., Leliaert F., Wetherbee R., De Clerck O., Duchene S., Steenwyk J.L., & Verbruggen H.<br />
		<b>Stepwise evolutionary transitions in Cladophorales plastid genomes reveals origins of hairpin chromosomes.</b><br />
		Submitted <br />
	</td></tr>
	<tr><td>
		Tong Y., Marcelino V.R., Turnbull R. & Verbruggen H.<br />
		<b>ChloroScan: Recovering plastid genome bins from metagenomic data.</b><br />
		Submitted <br />
		Preprint: <a href="https://doi.org/10.48550/arXiv.2510.10950">arXiv</a>
	</td></tr>
	<tr><td>
		Hossen R., Bjornson S., Bringloe T. & Verbruggen H.<br />
		<b>Extensive nuclear datasets resolve the phylogeny of siphonous green algae and identify genome duplications as a contributing factor to evolutionary adaptations.</b><br />
		Submitted <br />
	</td></tr>
	<tr><td>
		Hastings A., Cremen C., Courtney M., Du Y. & Verbruggen H.<br />
		<b>Enhancing species discovery and description in algal turfs: a case study in the green alga <i>Pseudoderbesia</i> (Bryopsidales).</b><br />
		Submitted <br />
	</td></tr>
	<tr><td>
		Waskowicz A., Verbruggen H. & Diaz-Tapia P.<br />
		<b>Taxonomic revisions of <i>Dipterosiphonia</i> (Rhodomelaceae, Rhodophyta) in southeastern Australia reveal <i>D. erecta</i> sp. nov.</b><br />
		Submitted <br />
	</td></tr>
	<tr><td>
		Salazar V.W., Verbruggen H., Marcelino V.R. & Le Cao K.A.<br />
		<b>Global picoplankton biogeography revealed by metagenomic and climatic data integration.</b><br />
		Submitted <br />
		Preprint: <a href="https://doi.org/10.1101/2024.11.23.624595">bioRxiv</a>
	</td></tr>
	<tr><td>
		Turnbull R., Steenwyk J., Mutch S.J., Scholten P., Salazar V.W., Birch J. & Verbruggen H.<br />
		<b>OrthoFlow: phylogenomic analysis and diagnostics with one command.</b><br />
		Submitted <br />
		Preprint: <a href="https://doi.org/10.21203/rs.3.rs-3699210/v1">ResearchSquare</a>
	</td></tr>
	<tr><td>
		Brito J., Verbruggen H., Cassano V.. & Diaz-Tapia P.<br />
		<b>Morphological and molecular approach uncovers hidden cryptic diversity within <i>Polysiphonia sertularioides</i> species complex (Rhodomelaceae, Rhodophyta).</b><br />
		<i>Journal of Phycology</i>: In press <br />
		DOI: <a href="https://doi.org/10.1111/jpy.70096">10.1111/jpy.70096</a>
	</td></tr>
	<tr><td>
		Huisman J.M., Verbruggen H. & Saunders G.W.<br />
		<b>The Western Australian species of <i>Callithamnion</i> and <i>Corynospora</i> (Ceramiales, Rhodophyta) described by W.H.Harvey</b><br />
		<i>Australian Systematic Botany</i>: In press <br />
	</td></tr>
	<tr><td>
		Cottier-Cook E.J., Brodie J., Corrigan S., De Clerck O., Lim P.E., Maggs C.A., Msuya F.E., Pellizzari F., Liu T., Wernberg T., Verbruggen H., Wilbraham J. & Yokoya N.S.<br />
		<b>Section 4, Sub-chapter 4i: Macroalgae, not including phytoplankton</b><br />
		United Nations World Ocean Assessment: Submitted <br />
	</td></tr>


<!--
	<tr><td>
		Diaz-Tapia P., Bribiesca-Contreras G., Maggs C.A. & Verbruggen H.<br />
		<b>Evolutionary radiation of the red algal family Rhodomelaceae after the Permian-Triassic mass extinction.</b><br />
		Submitted <br />
	</td></tr>
-->
</table>



<h2>2025</h2>
<table>
	<tr><td>
		Verbruggen H., Uthanumallian K., Powrie F., Jalali T., Cremen C., Preuss M., Duchene S. & Diaz-Tapia P. (2025)<br />
		<b>Scaling up species delimitation from DNA barcodes to whole organelle genomes: strong evidence for discordance among genes and methods for the red alga <i>Dasyclonium</i>.</b><br />
		<i>Molecular Ecology Resources</i> 25: e14132<br />
		DOI: <a href="https://doi.org/10.1111/1755-0998.14132">1755-0998.14132</a>
	</td></tr>
	<tr><td>
		Terpis K., Salomaki E., Barcyte D., Panek T., Verbruggen H., Kolisko M., Bailey J.C., Elias M, & Lane C. (2025)<br />
		<b>Multiple plastid losses within photosynthetic stramenopiles revealed by comprehensive phylogenomics.</b><br />
		<i>Current Biology</i> 35: 483-499<br />
		DOI: <a href="https://doi.org/10.1016/j.cub.2024.11.065">10.1016/j.cub.2024.11.065</a> &mdash; Preprint: <a href="https://www.biorxiv.org/content/10.1101/2024.02.03.578753v1">bioRxiv</a>
	</td></tr>
	<tr><td>
		Hsieh Y.E., Tandon K., Verbruggen H. & Nikoloski Z. (2025)<br />
		<b>Integration of metatranscriptomics data improves the predictive capacity of microbial community metabolic models.</b><br />
		<i>ISME Journal</i> 19: wraf109<br />
        DOI: <a href="https://doi.org/10.1093/ismejo/wraf109">10.1093/ismejo/wraf109</a>
	</td></tr>
	<tr><td>
		Wetherbee R., van de Meene A., Hossen R., Andersen R.A. & Verbruggen H. (2025)<br />
		<b>The Great Barrier Reef, a center for Pelagophyceae (Heterokontophyta) diversity, including a new genus and seven new species.</b><br />
		<i>Journal of Phycology</i> 61: 678-698<br />
		DOI: <a href="https://doi.org/10.1111/jpy.70030">10.1111/jpy.70030</a>
	</td></tr>
	<tr><td>
		Verbruggen H., Gong S., Yu K., Tandon K., Ricci F. & Liang J. (2025)<br />
		<b>Extensive diversity of <i>Symbiochlorum</i>-related algae from environmental sequences and culture strains supports the description of the new family Symbiochloraceae (Ulvophyceae, Chlorophyta).</b><br />
		<i>Journal of Phycology</i> 61: 926-932<br />
        DOI: <a href="https://doi.org/10.1111/jpy.70046">10.1111/jpy.70046</a>
	</td></tr>
	<tr><td>
		Bringloe T., Grant W., Zaparenkov D., Starko S., Fort A., Inaba M., Sulpice R., Saunders G.W. & Verbruggen H. (2025)<br />
		<b>Revisiting the species problem in northeast Pacific ribbon kelp lineages (genus <i>Alaria</i>): lessons learned using whole genome data.</b><br />
		<i>Journal of Phycology</i> 61: 777-791<br />
		DOI: <a href="https://doi.org/10.1111/jpy.70040">10.1111/jpy.70040</a>
	</td></tr>
	<tr><td>
		Huisman J.M. & Verbruggen H. (2025)<br />
		<b><i>Pseudobryopsis australis</i> (Ulvophyceae, Pseudobryopsidaceae), a new species of marine green algae from south-west Western Australia.</b><br />
		<i>Nuytsia</i> 36: 95–99<br />
		DOI: <a href="https://doi.org/10.58828/nuy01106">10.58828/nuy01106</a>
	</td></tr>
	<tr><td>
		Hossen R., Courtney M., Sim A., Khan M.A.A.K., Verbruggen H. & Bringloe T. (2025)<br />
		<b>An optimized CTAB method for genomic DNA extraction from green seaweeds (Ulvophyceae).</b><br />
		<i>Applications in Plant Sciences</i> 13: e11625 <br />
		DOI: <a href="https://doi.org/10.1002/aps3.11625">10.1002/aps3.11625</a>
	</td></tr>
	<tr><td>
		Tandon K., Hu J., Pasella M.M., Pushpakumara B.L.D.U. & Verbruggen H. (2025)<br />
		<b>Coral skeleton dwelling endolithic algae: <i>Ostreobium</i> and its biology.</b><br />
		In: Coral Reef Microbiome (Peixoto R. & Voolstra C.R., eds.), pp. 41-46<br />
	</td></tr>
</table>

<h2>2024</h2>
<table>
	<tr><td>
		Assis J., Fernández Bejarano S.J., Salazar V.W., Schepers L., Gouvea L., Fragkopoulou E., Leclercq F., Vanhoorne B., Tyberghein L., Serrao E.A., Verbruggen H. & De Clerck O. (2024)<br />
		<b>Bio-ORACLE v3.0. Pushing marine data layers to the CMIP6 Earth system models of climate change research</b><br />
		<i>Global Ecology and Biogeography</i> 33: e13813<br />
		DOI: <a href="https://doi.org/10.1111/geb.13813">10.1111/geb.13813</a>
	</td></tr>
	<tr><td>
		Uthanumallian K., Del Cortona A., Coelho S. De Clerck O., Duchene S. & Verbruggen H. (2024)<br />
		<b>Genome-wide patterns of selection-drift variation strongly associate with organismal traits across the green algae.</b><br />
		<i>Genome Research</i> 34: 1130-1139<br />
		DOI: <a href="https://doi.org/10.1101/gr.279002.124">10.1101/gr.279002.124</a>
	</td></tr>
	<tr><td>
		Denoeud F., Godfroy O., Cruaud C., Heesch S., Nehr Z., Tadrent N., Couloux A., Brillet-Gueguen L., Delage L., Mckeown D., Motomura T., Sussfeld D., Fan X., Mazeas L., Terrapon N., Barrera-Redondo J., Petroll R., Reynes L., Choi S.W., Jo J., Uthanumallian K., Bogaert K., Duc C., Ratchinski P., Lipinska A., Noel B., Murphy E.A., Lohr M., Khatei A., Hamon-Giraud P., Vieira C., Akerfors S.S., Akita S., Avia K., Badis Y., Barbeyron T., Belcour A., Berrabah W., Blanquart S., Bouguerba-Collin A., Bringloe T., Cattolico R.A., Cormier A., Cruz de Carvalho H., Dallet R., De Clerck O., Debit A., Denis E., Destombe C., Dinatale E., Dittami S., Drula E., Faugeron S., Got J., Graf L., Groisillier A., Guillemin M.L., Harms L., Hatchett W.J., Henrissat B., Hoarau G., Jollivet C., Jueterbock A., Kayal E., Knoll A.H., Kogame K., Le Bars A., Leblanc C., Le Gall L., Ley R., Liu X., LoDuca S.L., Lopez P.J., Lopez P., Manirakiza E., Massau K., Mauger S., Mest L., Michel G., Monteiro C., Nagasato C., Negre D., Pelletier E., Phillips N., Potin P., Rensing S.A., Rousselot E., Rousvoal S., Schroeder D., Scornet D., Siegel A., Tirichine L., Tonon T., Valentin K., Verbruggen H., Weinberger F., Wheeler G., Kawai H., Peters A.F., Yoon H.S., Herve C., Ye N., Bapteste E., Valero M., Markov G.V., Corre E., Coelho S.M., Wincker P., Aury J.M. & Cock J.M. (2024)<br />
		<b>Evolutionary genomics of the emergence of brown algae as key components of coastal ecosystems.</b><br />
		<i>Cell</i> 187: 6943-6965 <br />
		DOI: <a href="https://doi.org/10.1016/j.cell.2024.10.049">10.1016/j.cell.2024.10.049</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2024.02.19.579948">bioRxiv</a>
	</td></tr>
		<tr><td>
		Diaz-Tapia P. & Verbruggen H. (2024)<br />
		<b>Resolving the taxonomy of the <i>Polysiphonia scopulorum</i> complex and the <i>Bryocladia</i> lineage (Rhodomelaceae, Rhodophyta).</b><br />
		<i>Journal of Phycology</i> 60: 49-72<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13402">10.1111/jpy.13402</a>
	</td></tr>
	<tr><td>
		Hsieh Y.E., Tandon K., Verbruggen H. & Nikoloski Z. (2024)<br />
		<b>Comparative analysis of metabolic models of microbial communities reconstructed from automated tools and consensus approaches.</b><br />
		<i>npj Systems Biology and Applications</i> 10: 54<br />
		DOI: <a herf="https://doi.org/10.1038/s41540-024-00384-y">10.1038/s41540-024-00384-y</a>
	</td></tr>
	<tr><td>
		Bjornson S., Verbruggen H., Upham N., & Steenwyk J. (2024)<br />
		<b>Reticulate evolution: Detection and utility in the phylogenomics era.</b><br />
		<i>Molecular Phylogenetics and Evolution</i> 201: 108197 <br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2024.108197">10.1016/j.ympev.2024.108197</a>
	</td></tr>
	<tr><td>
		Wetherbee R., Bringloe T.T., Leliaert F. & Verbruggen H. (2024)<br />
		<b>Molecular data and culture observations show that the microfilamentous marine alga <i>Uronema marinum</i> is a member of the genus <i>Okellya</i> (Cladophorales, Chlorophyta).</b><br />
		<i>Cryptogamie Algologie</i> 45: 31-37<br />
		PDF: <a href="https://sciencepress.mnhn.fr/sites/default/files/articles/pdf/algologie2024v45a3.pdf">download</a>
	</td></tr>
	<tr><td>
		Verbruggen H. & Ballesteros E. (2024)<br />
		<b>An Atlantic origin for the introduced species <i>Halimeda incrassata</i>.</b><br />
		<i>Mediterranean Marine Science</i> 25: 55-57<br />
		DOI: <a href="https://doi.org/10.12681/mms.35993">10.12681/mms.35993</a>
	</td></tr>
	<tr><td>
		Huisman J.M., Verbruggen H., Hossen R., Rybalka N. & Entwisle T.J. (2024)<br />
		<b>Morphological and molecular analyses of <i>Vaucheria</i> section <i>Piloboloideae</i> (Xanthophyceae: Vaucheriaceae) indicate alternative species identities for broadly distributed taxa.</b><br />
		<i>Phycologia</i> 63: 170-178<br />
		DOI: <a href="https://doi.org/10.1080/00318884.2024.2306656">10.1080/00318884.2024.2306656</a>
	</td></tr>
</table>



<h2>2023</h2>
<table>
	<tr><td>
		Salazar V.W., Shaban B., del Mar Quiroga M., Turnbull R., Tescari E., Marcelino V.R., Verbruggen H. & Le Cao K.A. (2023)<br />
		<b>Metaphor - A workflow for streamlined assembly and binning of metagenomes.</b><br />
		<i>GigaScience</i> 12: giad055<br />
		DOI: <a href="https://doi.org/10.1093/gigascience/giad055">10.1093/gigascience/giad055</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2023.02.09.527784">bioRxiv</a>
	</td></tr>
	<tr><td>
		Ricci F., Tandon K., Mosshammer M., Cho E.H.J., Blackall L.L., K&uuml;hl M. & Verbruggen H. (2023)<br />
		<b>Fine-scale mapping of physicochemical and microbial landscapes of the coral skeleton.</b><br />
		<i>Environmental Microbiology</i> 25: 1505-1521<br />
		DOI: <a href="https://doi.org/10.1111/1462-2920.16369">10.1111/1462-2920.16369</a> &mdash; Preprint: <a href="https://doi.org/10.21203/rs.3.rs-1735748/v1">ResearchSquare</a>
	</td></tr>
	<tr><td>
		Tandon K., Pasella M.M., Iha C., Ricci F., Hu J., O'Kelly C.J., Medina M., K&uuml;hl M. & Verbruggen H. (2023)<br />
		<b>Every refuge has its price: <i>Ostreobium</i> as a model for understanding how algae can live in rock and stay in business.</b><br />
		<i>Seminars in Cell & Developmental Biology</i> 134: 27-36<br />
		DOI: <a href="https://doi.org/10.1016/j.semcdb.2022.03.010">10.1016/j.semcdb.2022.03.010</a>
	</td></tr>
	<tr><td>
		Tandon K., Ricci F., Costa J., Medina M., K&uuml;hl M., Blackall L.L. & Verbruggen H. (2023)<br />
		<b>Genomic view of the diversity and functional role of archaea and bacteria in the skeleton of the reef-building corals <i>Porites lutea</i> and <i>Isopora palifera</i>.</b><br />
		<i>GigaScience</i> 12: giac127<br />
		DOI: <a href="https://doi.org/10.1093/gigascience/giac127">10.1093/gigascience/giac127</a> &mdash; Preprint: <a href="https://doi.org/10.21203/rs.3.rs-1890655/v1">ResearchSquare</a>
	</td></tr>
	<tr><td>
		Pushpakumara B.L.D.U., Tandon K., Willis A. & Verbruggen H. (2023)<br />
		<b>Unravelling microalgal-bacterial interactions in aquatic ecosystems through 16S rRNA gene-based co-occurrence networks.</b><br />
		<i>Scientific Reports</i> 13: 2743 <br />
		DOI: <a href="https://doi.org/10.1038/s41598-023-27816-9">10.1038/s41598-023-27816-9</a> &mdash; 
		Preprint: <a href="https://doi.org/10.1101/2022.01.05.475147">bioRxiv</a>
	</td></tr>
	<tr><td>
		Wetherbee R., Bringloe T.T., van de Meene A., Andersen R.A. & Verbruggen H. (2023)<br />
		<b>Structure and formation of the perforated theca defining the Pelagophyceae (Heterokonta), and three new genera that substantiate the diverse nature of the class.</b><br />
		<i>Journal of Phycology</i> 59: 126-151<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13294">10.1111/jpy.13294</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Rodriguez-Bujan I. Maggs C.A. & Verbruggen H. (2023)<br />
		<b>Phylogenomic analysis of pseudocryptic diversity reveals the new genus <i>Deltalsia</i> (Rhodomelaceae, Rhodophyta).</b><br />
		<i>Journal of Phycology</i> 59: 264-276<br />
		DOI: <a href="http://doi.org/10.1111/jpy.13311">10.1111/jpy.13311</a> 
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Nelson W.A. & Verbruggen H. (2023)<br />
		<b>Molecular analysis of turf algae reveal a new species and an undetected introduction in the Pterosiphonieae (Rhodomelaceae, Rhodophyta).</b><br />
		<i>Journal of Phycology</i> 59: 603-618<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13336">10.1111/jpy.13336</a>
	</td></tr>
	<tr><td>
		Pushpakumara B.L.D.U., Tandon K., Willis A. & Verbruggen H. (2023)<br />
		<b>The bacterial microbiome of the coral skeleton algal symbiont <i>Ostreobium</i> shows preferential associations and signatures of phylosymbiosis.</b><br />
		<i>Microbial Ecology</i> 86: 2032-2046<br />
		DOI: <a href="https://doi.org/10.1007/s00248-023-02209-7">10.1007/s00248-023-02209-7</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2022.12.13.520198">bioRxiv</a>
	</td></tr>
	<tr><td>
		Shaban B., del Mar Quiroga M., Turnbull R., Tescari E., Le Cao K.A. & Verbruggen H. (2023)<br />
		<b>MetaGenePipe: an automated, portable pipeline for contig-based functional and taxonomic analysis.</b><br />
		<i>Journal of Open Source Software</i> 8: 4851<br />
		DOI: <a href="https://doi.org/10.21105/joss.04851">10.21105/joss.04851</a> &mdash; Preprint: <a href="https://doi.org/10.21203/rs.3.rs-1999920/v1">ResearchSquare</a>
	</td></tr>
	<tr><td>
		West J.A., Zuccarello G.C., Verbruggen H., Powling J., Miller J. & Loiseaux-de Goer S. (2023)<br />
		<b><i>Amoenothamnion planktonicum</i> (Heterothamnieae, Ceramiaceae, Rhodophyta), an annual drift event along sandy beaches of southern Australia and its affinities.</b><br />
		<i>Notulae Algarum</i> 271: 1-7<br />
		PDF download: <a href="https://notulaealgarum.org/2023/documents/Notulae%20Algarum%20No.%20271.pdf">Notulae Algarum</a>
	</td></tr>
	<tr><td>
		Preuss M., Diaz-Tapia P., Verbruggen H. & Zuccarello G.C. (2023)<br />
		<b>Gene-rich plastid genomes of two parasitic red algal species, <i>Laurencia australis</i> and <i>L. verruciformis</i> (Rhodomelaceae, Ceramiales), and a taxonomic revision of <i>Janczewskia</i>.</b><br />
		<i>Journal of Phycology</i> 59: 950-962<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13373">10.1111/jpy.13373</a>
	</td></tr>
	<tr><td>
		Huisman J.M. & Verbruggen H. (2023)<br />
		<b>A morphological and molecular study supports the recognition of <i>Rhipilia psammophila</i> sp. nov. and <i>Rhipilia baculifera</i> comb. nov. (Halimedaceae, Chlorophyta) from southern Australia.</b><br />
		<i>Australian Systematic Botany</i> 36: 427-436<br />
		DOI: <a href="https://doi.org/10.1071/SB23016">10.1071/SB23016</a>
	</td></tr>
</table>

<h2>2022</h2>
<table>
	<tr><td>
		Uthanumallian K., Iha C., Repetti S.I., Chan C.X., Bhattacharya D., Duchene S. & Verbruggen H. (2022)<br />
		<b>Tightly constrained genome reduction and relaxation of purifying selection during secondary plastid endosymbiosis.</b><br />
		<i>Molecular Biology and Evolution</i> 39: msab295<br />
		DOI: <a href="https://doi.org/10.1093/molbev/msab295">10.1093/molbev/msab295</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2021.05.27.446077">bioRxiv</a>
	</td></tr>
	<tr><td>
		Repetti S.I., Iha C., Uthanumallian K., Jackson C.J., Chen Y., Chan C.X. & Verbruggen H. (2022)<br />
		<b>Nuclear genome of a pedinophyte pinpoints genomic innovation and streamlining in the green algae.</b><br />
		<i>New Phytologist</i> 233: 2144-2154<br />
		DOI: <a href="https://doi.org/10.1111/nph.17926">10.1111/nph.17926</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2021.10.04.463119">bioRxiv</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Wilkinson D.P., Goldsmit J., Savoie A.M., Filbee-Dexter K., Macgregor K.A., Howland K.L., McKindsey C.W. & Verbruggen H. (2022)<br />
		<b>Arctic marine forest distribution models showcase potentially severe habitat losses for cryophilic species under climate change.</b><br />
		<i>Global Change Biology</i> 28: 3711-3727 <br />
		DOI: <a href="https://doi.org/10.1111/gcb.16142">10.1111/gcb.16142</a>
	</td></tr>
	<tr><td>
		Ricci F., Tandon K., Black J.R., Le Cao K.A., Blackall L.L. & Verbruggen H. (2022)<br />
		<b>Host traits and phylogeny contribute to shaping coral-bacterial symbioses.</b><br />
		<i>mSystems</i> 7: e00044-22 <br />
		DOI: <a href="https://doi.org/10.1128/msystems.00044-22">10.1128/msystems.00044-22</a> &mdash; Preprint: <a href="https://doi.org/10.21203/rs.3.rs-757354/v1">Research Square</a>
	</td></tr>
	<tr><td>
		Pasella M.M., Lee M.F.E., Marcelino V.R., Willis A. & Verbruggen H. (2022)<br />
		<b>Ten <i>Ostreobium</i> strains from Great Barrier Reef corals as a resource for algal endolith biology and genomics.</b><br />
		<i>Phycologia</i> 61: 452-458<br />
		DOI: <a href="https://doi.org/10.1080/00318884.2022.2064132">10.1080/00318884.2022.2064132</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2021.08.16.453452">bioRxiv</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Fort A., Inaba M., Sulpice R., Ghriofa C.N., Mols-Mortensen A., Filbee-Dexter K., Vieira C., Kawai H., Hanyuda T., Krause-Jensen D., Olesen B., Starko S. & Verbruggen H. (2022)<br />
		<b>Whole genome population structure of North Atlantic kelp confirms high-latitude glacial refugia.</b><br />
		<i>Molecular Ecology</i> 31: 6473-6488<br />
		DOI: <a href="https://doi.org/10.1111/mec.16714">10.1111/mec.16714</a>
	</td></tr>
</table>

<h2>2021</h2>
<table>
	<tr><td>
		Iha C., Dougan K.E., Varela J.A., Avila V., Jackson C.J., Bogaert K.A., Chen Y., Judd L.M., Wick R., Holt K.E., Pasella M.M., Ricci F., Repetti S.I., Medina M., Marcelino V.R., Chan C.X. & Verbruggen H. (2021)<br />
		<b>Genomic adaptations to an endolithic lifestyle in the coral-associated alga <i>Ostreobium</i>.</b><br />
		<i>Current Biology</i> 31: 1393-1402<br />
		DOI: <a href="https://doi.org/10.1016/j.cub.2021.01.018">10.1016/j.cub.2021.01.018</a> &mdash; Preprint: <a href="https://doi.org/10.1101/2020.07.21.211367">bioRxiv</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Sauermann R., Krause-Jensen D., Olesen B., Klimova A., Klochkova T.A. & Verbruggen H. (2021)<br />
		<b>High-througput sequencing of the kelp <i>Alaria</i> (Phaeophyceae) reveals novel epi-endobiotic associations, including a likely phaeophycean parasite.</b><br />
		<i>European Journal of Phycology</i> 56: 494-504<br />
		DOI: <a href="https://doi.org/10.1080/09670262.2021.1882704">10.1080/09670262.2021.1882704</a>
	</td></tr>
	<tr><td>
		Ricci F., Fordyce A., Leggat W., Blackall L.L., Ainsworth T. & Verbruggen H. (2021)<br />
		<b>Multiple techniques point to oxygenic phototrophs dominating the <i>Isopora palifera</i> skeletal microbiome.</b><br />
		<i>Coral Reefs</i> 40: 275-282<br />
		DOI: <a href="https://doi.org/10.1007/s00338-021-02068-z">10.1007/s00338-021-02068-z</a>
	</td></tr>
	<tr><td>
		Vieira C., Steen F., D'hondt S., Bafort Q., Tyberghein L., Fernandez-Garcia C., Wysor B., Tronholm A., Mattio L., Payri C., Kawai H., Saunders G., Leliaert F., Verbruggen H. & De Clerck O. (2021)<br />
		<b>Global biogeography and diversification of a group of brown seaweeds (Phaeophyceae) driven by clade-specific evolutionary processes.</b><br />
		<i>Journal of Biogeography</i> 48: 703-715<br />
		DOI: <a href="https://doi.org/10.1111/jbi.14047">10.1111/jbi.14047</a>
	</td></tr>
	<tr><td>
		Wetherbee R., Bringloe T.T., Costa J.F., van de Meene A., Andersen R.A. & Verbruggen H. (2021)<br />
		<b>New pelagophytes show a novel mode of algal colony development and reveal a perforated theca that may define the class.</b><br />
		<i>Journal of Phycology</i> 57: 396-411<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13074">10.1111/jpy.13074</a>
	</td></tr>
	<tr><td>
		Zou X., Verbruggen H., Li T., Zhu J., Chen Z., He W., Bao S. & Sun J. (2021)<br />
		<b>Identification of polycistronic transcriptional units and non-canonical introns in green algal chloroplasts based on long-read RNA sequencing data.</b><br />
		<i>BMC Genomics</i> 22: 298 <br />
		DOI: <a href="https://doi.org/10.1186/s12864-021-07598-y">10.1186/s12864-021-07598-y</a>
	</td></tr>
	<tr><td>
		Preuss M., Verbruggen H., West J. & Zuccarello G.C. (2021)<br />
		<b>Divergence times and plastid phylogenomics within the intron-rich order Erythropeltales (Compsopogonophyceae, Rhodophyta).</b><br />
		<i>Journal of Phycology</i> 57: 1035-1044<br/>
		DOI: <a href="https://doi.org/10.1111/jpy.13159">10.1111/jpy.13159</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Zaparenkov D., Starko S., Grant W., Vieira C., Kawai H., Hanyuda T., Filbee-Dexter K., Klimova A., Klochkova T., Krause-Jensen D., Olesen B. & Verbruggen H. (2021)<br />
		<b>Whole genome sequencing reveals forgotten lineages and recurrent hybridizations within the kelp genus <i>Alaria</i> (Phaeophyceae).</b><br />
		<i>Journal of Phycology</i> 57: 1721-1738<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13212">10.1111/jpy.13212</a>
	</td></tr>
	<tr><td>
		Fang J., Wang H., Chen Y., Liu G., Verbruggen H. & Zhu H. (2021)<br />
		<b>Chloroplast genome traits correlate with organismal complexity and ecological traits in green algae.</b><br />
		<i>Frontiers in Ecology and Evolution</i> 9: 882 <br />
		DOI: <a href="https://doi.org/10.3389/fevo.2021.791166">10.3389/fevo.2021.791166</a>
	</td></tr>
	<tr><td>
		Fang J., Liu B., Wang H., Liu G., Verbruggen H. & Zhu H. (2021)<br />
		<b>Six newly sequenced chloroplast genomes from Trentepohliales: inflated genomes, alternative genetic code and dynamic evolution.</b><br />
		<i>Frontiers in Plant Science</i> 12: 2865<br />
		DOI: <a href="https://doi.org/10.3389/fpls.2021.780054">10.3389/fpls.2021.780054</a>
	</td></tr>
	<tr><td>
		Tonicelli G., Croce M.E., Diaz-Tapia P., Fredericq S., Freshwater D.W., Gauna M.C., Parodi E.R., Verbruggen H. & Hommersand M. (2021)<br />
		<b><i>Meridionella</i> gen. nov., a new genus of Cystocloniaceae (Gigartinales, Rhodophyta) from the southern hemisphere including <i>M. obtusangula</i> comb. nov. and <i>M. antarctica</i> sp. nov.</b><br />
		<i>Journal of Phycology</i> 57: 797-816<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13127">10.1111/jpy.13127</a>
	</td></tr>
</table>


<h2>2020</h2>
<table>
	<tr><td>
		del Cortona A., Jackson C.J., Bucchini F., Van Bel M., D'Hondt S., Skaloud P., Delwiche C.F., Knoll A.H., Raven J.A., Verbruggen H., Vandepoele K., De Clerck O. & Leliaert F. (2020)<br />
		<b>Neoproterozoic origin and multiple transitions to macroscopic growth in green seaweeds.</b><br />
		<i>Proceedings of the National Academy of Sciences of the USA</i> 117: 2551-2559<br />
		DOI: <a href="https://doi.org/10.1073/pnas.1910060117">10.1073/pnas.1910060117</a> &mdash; preprint: <a href="https://doi.org/10.1101/668475">10.1101/668475</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Verbruggen H. & Saunders G.W. (2020)<br />
		<b>Unique biodiversity in Arctic marine forests is shaped by diverse recolonization pathways and far northern glacial refugia.</b><br />
		<i>Proceedings of the National Academy of Sciences of the USA</i> 117: 22590-22596<br />
		DOI: <a href="https://doi.org/10.1073/pnas.2002753117">10.1073/pnas.2002753117</a> &mdash; preprint: <a href="https://doi.org/10.1101/2020.03.19.999466">bioRxiv</a>
	</td></tr>
	<tr><td>
		Bringloe T.T., Starko S., Wade R.M., Vieira C., Kawai H., De Clerck O., Cock J.M., Coelho S.M., Destombe C., Valero M., Neiva J., Pearson G.A., Faugeron S., Serrao E.A. & Verbruggen H. (2020)<br />
		<b>Phylogeny and evolution of the brown algae.</b><br />
		<i>Critical Reviews in Plant Sciences</i> 39: 281-321<br />
		DOI: <a href="https://doi.org/10.1080/07352689.2020.1787679">10.1080/07352689.2020.1787679</a>
	</td></tr>
	<tr><td>
		Repetti S.I., Jackson C.J., Judd L.M., Wick R.R., Holt K.E. & Verbruggen H. (2020)<br />
		<b>The inflated mitochondrial genomes of siphonous green algae reflect processes driving expansion of noncoding DNA and proliferation of introns.</b><br />
		<i>PeerJ</i> 8: e8273<br />
		DOI: <a href="https://doi.org/10.7717/peerj.8273">10.7717/peerj.8273</a>
	</td></tr>
	<tr><td>
		Goes-Neto A., Marcelino V.R., Verbruggen H., Ferreira da Silva F. & Badotti F. (2020)<br />
		<b>Fungal diversity of coral skeletons in the southwestern Pacific using 18S rDNA metabarcoding. </b><br />
		<i>Coral Reefs</i> 39: 229-238<br />
		DOI: <a href="https://doi.org/10.1007/s00338-019-01880-y">10.1007/s00338-019-01880-y</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Ly M. & Verbruggen H. (2020)<br />
		<b>Extensive cryptic diversity in the widely distributed <i>Polysiphonia scopulorum</i> (Rhodomelaceae, Rhodophyta): molecular species delimitation and morphometric analyses.</b><br />
		<i>Molecular Phylogenetics and Evolution</i> 152: 106909<br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2020.106909">10.1016/j.ympev.2020.106909</a>
	</td></tr>
	<tr><td>
		Preuss M., Verbruggen H. & Zuccarello G.C. (2020)<br />
		<b>The organelle genomes in the photosynthetic red algal parasite <i>Pterocladiophila hemisphaerica</i> have elevated substitution rates and extreme gene loss in the plastid genome.</b><br />
		<i>Journal of Phycology</i> 56: 1006-1018 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12996">10.1111/jpy.12996</a>
	</td></tr>
	<tr><td>
		Charon J., Marcelino V.R., Wetherbee R., Verbruggen H. & Holmes E.C. (2020)<br />
		<b>Metatranscriptomic identification of diverse and divergent RNA viruses associated with green and chlorarachniophyte algae.</b><br />
		<i>Viruses</i> 12: 1180 <br />
		DOI: <a href="https://doi.org/10.3390/v12101180">10.3390/v12101180</a> &mdash; Preprint: <a href="https://biorxiv.org/cgi/content/short/2020.06.08.141184v1">bioRxiv</a>
	</td></tr>
	<tr><td>
		Graf L., Yang E.C., Han K.Y., Kupper F., Benes K.M., Oyadomari J., Herbert R.J.H., Verbruggen H., Wetherbee R., Andersen R.A. & Yoon H.S.<br />
		<b>Multigene phylogeny, morphological observation and re-examination of the literature lead to the description of the Phaeosacciophyceae classis nova and four new species of the Heterokontophyta SI clade.</b><br />
		<i>Protist</i> 171: 125781 <br />
		DOI: <a href="https://doi.org/10.1016/j.protis.2020.125781">10.1016/j.protis.2020.125781</a>
	</td></tr>
	<tr><td>
		Rindi F., Pasella M.M., Lee M-F.E. & Verbruggen H. (2020)<br />
		<b>Phylogeography of the Mediterranean green seaweed <i>Halimeda tuna</i>.</b><br />
		<i>Journal of Phycology</i> 56: 1109-1113<br />
		DOI: <a href="https://doi.org/10.1111/jpy.13006">10.1111/jpy.13006</a>
	</td></tr>
	<tr><td>
		Pineiro-Corbeira C., Verbruggen H. & Diaz-Tapia P. (2020)<br />
		<b>Molecular survey of the red algal family Rhodomelaceae (Ceramiales, Rhodophyta) in Australia reveals new introduced species.</b><br />
		<i>Journal of Applied Phycology</i> 32: 2535-2547<br />
		DOI: <a href="https://doi.org/10.1007/s10811-019-01932-4">10.1007/s10811-019-01932-4</a>
	</td></tr>
	<tr><td>
		Schneider C.W., Lam D.W. & Verbruggen H. (2020)<br />
		<b>DNA sequencing and anatomy demonstrate that Pacific <i>Codium simulans</i> is a genetically variable species found in the floras of Bermuda and Florida. </b><br />
		<i>Phycological Research</i> 68: 98-102<br />
		DOI: <a href="https://doi.org/10.1111/pre.12396">10.1111/pre.12396</a>
	</td></tr>
	<tr><td>
		Lagourgue L., Verbruggen H., Ampou E.E. & Payri C.E. (2020)<br />
		<b>One hundred years later, resurrection of <i>Tydemania gardineri</i> (Udoteaceae, Chlorophyta) based on molecular and morphological data. </b><br />
		<i>European Journal of Phycology</i> 55: 89-99<br />
		DOI: <a href="https://doi.org/10.1080/09670262.2019.1654618">10.1080/09670262.2019.1654618</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Maggs C.A., Nelson W., Macaya E.C. & Verbruggen H. (2020)<br />
		<b>Reassessment of the genus <i>Lophurella</i> (Rhodomelaceae, Rhodophyta) from Australia and New Zealand reveals four cryptic species. </b><br />
		<i>European Journal of Phycology</i> 55: 113-128<br />
		DOI: <a href="https://doi.org/10.1080/09670262.2019.1659419">10.1080/09670262.2019.1659419</a>
	<tr><td>
		Huisman J.M. & Verbruggen H. (2020)<br />
		<b><i>Pseudoderbesia eckloniae</i>, sp. nov. (Bryopsidaceae, Ulvophyceae) from Western Australia.</b><br />
		<i>Cryptogamie Algologie</i> 41: 19-23 <br />
		DOI: <a href="https://doi.org/10.5252/cryptogamie-algologie2020v41a3">10.5252/cryptogamie-algologie2020v41a3</a>
	</td></tr>
	<tr><td>
		Dumilag R.V., Dumago F.S., Cabudoy R.K.R., Peralta M.C.E., Li C.C., Gamus G.C.V., Romero R.G.T., Yap S.L., Roleda M.Y., Geraldino P.J.L., Verbruggen H., Leliaert F., Draisma S.G.A., Liao L.M. & Kraft G.T. (2020)<br />
		<b>The Ulvophyceae (Chlorophyta) of eastern Sorsogon, Philippines, including <i>Halimeda magnicuneata</i> sp. nov. (Bryopsidales).</b><br />
		<i>Botanica Marina</i> 63: 439-453<br />
		DOI: <a href="https://doi.org/10.1515/bot-2020-0017">10.1515/bot-2020-0017</a>
	</td></tr>
</table>





<h2>2019</h2>
<table>
	<tr><td>
		Ricci F., Marcelino V.R., Blackall L.L., K&uuml;hl M., Medina M. & Verbruggen H. (2019) <br />
		<b>Beneath the surface: community assembly and functions of the coral skeleton microbiome.</b><br />
		<i>Microbiome</i> 7: 159 <br />
		DOI: <a href="https://doi.org/10.1186/s40168-019-0762-y">10.1186/s40168-019-0762-y</a>
	</td></tr>
	<tr><td>
		Wetherbee R., Jackson C.J., Repetti S.I., Clementson L.A., Costa J.F., van de Meene A., Crawford S. & Verbruggen H. (2019) <br />
		<b>The golden paradox &ndash; a new heterokont lineage with chloroplasts surrounded by two membranes. </b> <br />
		<i>Journal of Phycology</i> 55:  257-278<br />
		DOI: <a href="https://doi.org/10.1111/jpy.12822">10.1111/jpy.12822</a>
	</td></tr>
	<tr><td>
		Cremen M.C.M., Leliaert F., West J., Lam D.W., Shimada S., Lopez-Bautista J. & Verbruggen H. (2019) <br />
		<b>Re-assessment of the classification of Bryopsidales (Chlorophyta) based on chloroplast phylogenomic analyses. </b><br />
		<i>Molecular Phylogenetics and Evolution</i> 130: 397-405<br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2018.09.009">10.1016/j.ympev.2018.09.009</a>
	</td></tr>
	<tr><td>
		Bribiesca-Contreras G., Pineda-Enriquez T., Marquez-Borras F., Solis-Marin F.A., Verbruggen H., Hugall A.F. & O'Hara T. (2019)<br />
		<b>Dark offshoot: Phylogenomic data sheds light on the evolutionary history of a new species of cave brittle star.</b><br />
		<i>Molecular Phylogenetics and Evolution</i> 136: 151-163<br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2019.04.014">10.1016/j.ympev.2019.04.014</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Pasella M.M., Verbruggen H. & Maggs C.A. (2019)<br />
		<b>Morphological evolution and classification of the red algal order Ceramiales inferred using plastid phylogenomics.</b><br />
		<i>Molecular Phylogenetics and Evolution</i> 137: 76-85<br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2019.04.022">10.1016/j.ympev.2019.04.022</a>
	</td></tr>
	<tr><td>
		Evans J.R., St. Amour N., Verbruggen H., Salomaki E.D. & Vis M.L. (2019)<br />
		<b>Chloroplast and mitochondrial genomes of <i>Balbiania investiens</i> (Balbianiales, Nemaliophycidae). </b><br />
		<i>Phycologia</i> 58: 310-318 <br />
		DOI: <a href="https://doi.org/10.1080/00318884.2019.1573349">10.1080/00318884.2019.1573349</a>
	</td></tr>
	<tr><td>
		Pasella M.M., Verbruggen H., Nelson W.A. & Diaz-Tapia P. (2019)<br />
		<b>The phylogenetic position of the morphologically unusual <i>Pleurostichidium falkenbergii</i> (Rhodomelaceae, Rhodophyta) based on plastid phylogenomics. </b><br />
		<i>Phycologia</i> 58: 319-325<br />
		DOI: <a href="https://doi.org/10.1080/00318884.2019.1574462">10.1080/00318884.2019.1574462</a>
	</td></tr>
	<tr><td>
		Bribiesca-Contreras G., Verbruggen H., Hugall A.F. & O'Hara T. (2019)<br />
		<b>Global biogeographic structuring of tropical shallow-water brittle stars. </b><br />
		<i>Journal of Biogeography</i> 46: 1287-1299 <br />
		DOI: <a href="https://doi.org/10.1111/jbi.13620">10.1111/jbi.13620</a>
	</td></tr>
	<tr><td>
		Wetherbee R., Marcelino V.R., Costa J.F., Grant B., Crawford S., Waller R.F., Andersen R.A., Berry D., McFadden G.I. & Verbruggen H. (2019)<br />
		<b>A new marine prasinophyte genus alternates between a flagellate and a dominant benthic stage with microrhizoids for adhesion.</b><br />
		<i>Journal of Phycology</i> 55: 1210-1225<br />
		DOI: <a href="https://doi.org/10.1111/jpy.12909">10.1111/jpy.12909</a>
	</td></tr>
</table>





<h2>2018</h2>
<table>
	<tr><td>
		Diaz-Tapia P., Maggs C.A., Macaya E.C. & Verbruggen H. (2018) <br />
		<b>Widely distributed red algae often represent hidden introductions, complexes of cryptic species or species with strong phylogeographic structure.</b><br />
		<i>Journal of Phycology</i> 54: 829-839 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12778">10.1111/jpy.12778</a>
	</td></tr>
	<tr><td>
		Iha C., Grassa C.J., Lyra G.M., Davis C.C., Verbruggen H. & Oliveira M.C. (2018) <br />
		<b>Organellar genomics: a useful tool to study evolutionary relationships and molecular evolution in Gracilariaceae (Rhodophyta).</b><br />
		<i>Journal of Phycology</i> 54: 775-787 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12765">10.1111/jpy.12765</a>
	</td></tr>
	<tr><td>
		Croce M.E., Diaz-Tapia P. & Verbruggen H. (2018) <br />
		<b>Molecular systematics of the siphonous green algal genus <i>Codium</i> in eastern Victoria, Australia. </b><br />
		<i>Cryptogamie Algologie</i> 39:391-407 <br />
		DOI: <a href="https://doi.org/10.7872/crya/v39.iss4.2018.391">10.7872/crya/v39.iss4.2018.39</a>
	</td></tr>
	<tr><td>
		Leliaert F., Payo D.A., Gurgel C.F.D., Schils T., Draisma S.G.A., Saunders G.W., Kamiya M., Sherwood A.R., Lin S.M., Huisman J.M., Le Gall L., Anderson R.J., Bolton J.J., Mattio L., Zubia M., Payri C., Spokes T., Johnston E.T., Vieira C., Coppejans E., D'hondt S., Verbruggen H. & De Clerck O. (2018) <br />
		<b>Patterns and drivers of species diversity in the Indo-Pacific red seaweed <i>Portieria</i>. </b><br />
		<i>Journal of Biogeography</i> 10: 2299-2313<br />
		DOI: <a href="https://doi.org/10.1111/jbi.13410">10.1111/jbi.13410</a>
	</td></tr>
	<tr><td>
		Yates K.L., Bouchet P.J., Caley M.J., Mengersen K., Randin C.F., Parnell S., Fielding A.H., Bamford A.J., Ban S., Barbosa A.M., Dormann C.F., Elith J., Embling C.B., Ervin G.N., Fisher R., Gould S., Graf R.F., Gregr E.J., Halpin P.N., Heikkinen R.K., Heinanen S., Jones A.R., Krishnakumar P.K., Lauria V., Lozano-Montes H., Mannocci L., Mellin C., Mesgaran M.B., Moreno-Amat E., Mormede S. Novaczek E., Oppel S., Ortuno Crespo G., Peterson A.T., Rapacciuolo G., Roberts J.J., Ross R.E. Scales K.L., Schoeman D., Snelgrove P., Sundblad G., Thuiller W., Torres L.G., Verbruggen H., Wang L., Wenger S., Whittingham M.J., Zharikov Y., Zurell D. & Sequeira A.M.M. (2018) <br />
		<b>Outstanding challenges in the transferability of ecological models.</b><br />
		<i>Trends in Ecology & Evolution</i> 33: 790-802 <br />
		DOI: <a href="https://doi.org/10.1016/j.tree.2018.08.001">10.1016/j.tree.2018.08.001</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Pasella M.M. & Verbruggen H. (2018)<br />
		<b>Molecular analyses resolve the phylogenetic position of <i>Polysiphonia adamsiae</i> (Rhodomelaceae, Rhodophyta) and reveal a strong phylogeographic structure in Australia.</b><br />
		<i>Phycologia</i> 57: 593-600 <br />
		DOI: <a href="https://doi.org/10.2216/18-36.1">10.2216/18-36.1</a>
	</td></tr>
	<tr><td>
		Oliveira M.C., Repetti S.I., Iha C., Jackson C.J., Diaz-Tapia P., Magalhaes K., Cassano V., Costa J.F., Cremen M.C.M., Marcelino V.R. & Verbruggen H. (2018) <br />
		<b>High-throughput sequencing for algal systematics.</b><br />
		<i>European Journal of Phycology</i> 53: 256-272 <br />
		DOI: <a href="https://doi.org/10.1080/09670262.2018.1441446">10.1080/09670262.2018.1441446</a>
	</td></tr>
	<tr><td>
		Cremen M.C.M., Leliaert F., Marcelino V.R. & Verbruggen H. (2018)  <br />
		<b>Large diversity of nonstandard genes and dynamic evolution of chloroplast genomes in siphonous green algae (Bryopsidales, Chlorophyta).</b>  <br />
		<i>Genome Biology and Evolution </i>10: 1048-1061 <br />
		DOI: <a href="https://doi.org/10.1093/gbe/evy063">10.1093/gbe/evy063</a>
	</td></tr>
	<tr><td>
		Paiano M.O., Del Cortona A., Costa J.F., Liu S.L., Verbruggen H., De Clerck O. & Necchi Jr O. (2018)  <br />
		<b>Complete mitochondrial genomes of six species of the freshwater red algal order Batrachospermales (Rhodophyta).</b> <br />
		<i>Mitochondrial DNA Part B - Resources </i>3: 607-610 <br />
		DOI: <a href="https://doi.org/10.1080/23802359.2018.1473734">10.1080/23802359.2018.1473734</a> 
	</td></tr>
	<tr><td>
		Assis J., Tyberghein L., Bosch S., Verbruggen H., Serrao E.A. & De Clerck O. (2018)  <br />
		<b>Bio-ORACLE v2.0: Extending marine data layers for bioclimatic modelling. </b> <br />
		<i>Global Ecology and Biogeography</i> 27: 277-284. <br />
		DOI: <a href="https://doi.org/10.1111/geb.12693">10.1111/geb.12693</a>
	</td></tr>
	<tr><td>
		Marcelino V.R., van Oppen M.J.H. & Verbruggen H. (2018)  <br />
		<b>Highly structured prokaryote communities exist within the skeleton of coral colonies.  </b><br />
		<i>ISME Journal</i> 12: 300-303 <br />
		DOI: <a href="https://doi.org/10.1038/ismej.2017.164">10.1038/ismej.2017.164</a>
	</td></tr>
	<tr><td>
		Jackson C., Knoll A.H., Chan C.X. & Verbruggen H. (2018)  <br />
		<b>Plastid phylogenomics with broad taxon sampling further elucidates the distinct evolutionary origins and timing of secondary green plastids.</b>  <br />
		<i>Scientific Reports</i> 8: 1523 <br />
		DOI: <a href="https://doi.org/10.1038/s41598-017-18805-w">10.1038/s41598-017-18805-w</a>
	</td></tr>
	<tr><td>
		Paiano M.O., Del Cortona A., Costa J.F., Liu S.L., Verbruggen H., De Clerck O. & Necchi O. (2018) <br /> 
		<b>Organization of plastid genomes in the freshwater red algal order Batrachospermales (Rhodophyta). </b> <br />
		<i>Journal of Phycology </i>54: 25-33 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12602">10.1111/jpy.12602</a>
	</td></tr>
	<tr><td>
		Hoffman R., Wynne M.J., Schils T., Lopez-Bautista J. & Verbruggen H. (2018)  <br />
		<b><i>Codium pulvinatum</i> (Bryopsidales, Chlorophyta), a new species from the Arabian Sea, recently introduced into the Mediterranean Sea.</b>  <br />
		<i>Phycologia</i> 57: 79-89 <br />
		DOI: <a href="https://doi.org/10.2216/17-74.1">10.2216/17-74.1</a>
	</td></tr>
</table>





<h2>2017</h2>
<table>
	<tr><td>
		del Cortona A., Leliaert F., Bogaert K.A., Turmel M., Boedeker C., Janouskovec J., Lopez-Bautista J.M., Verbruggen H., Vanepoele K., De Clerck O. (2017) <br />
		<b>The plastid genome in Cladophorales green algae is encoded by hairpin plasmids.  </b><br />
		<i>Current Biology</i> 27: 3771-3782 <br />	
		DOI: <a href="https://doi.org/10.1016/j.cub.2017.11.004">10.1016/j.cub.2017.11.004</a> &mdash; preprint: <a href="https://doi.org/10.1101/145037">10.1101/145037</a>
	</td></tr>
	<tr><td>
		Marcelino V.R., Morrow K.M., van Oppen M.J.H., Bourne D.G. & Verbruggen H. (2017)  <br />
		<b>Diversity and stability of coral endolithic microbial communities at a naturally high pCO2 reef. </b> <br />
		<i>Molecular Ecology</i> 26: 5334-5357 <br />
		DOI: <a href="https://doi.org/10.1111/mec.14268">10.1111/mec.14268</a>
	</td></tr>
	<tr><td>
		Bribiesca-Contreras G., Verbruggen H., Hugall A.F. & O'Hara T.D. (2017)  <br />
		<b>The importance of offshore origination revealed through ophiuroid phylogenomics.  </b><br />
		<i>Proceedings of the Royal Society B: Biological Sciences</i> 284: 20170160 <br />
		DOI: <a href="https://doi.org/10.1098/rspb.2017.0160">10.1098/rspb.2017.0160</a>
	</td></tr>
	<tr><td>
		Marcelino V.R. & Verbruggen H. (2017)  <br />
		<b>Reference datasets of <i>tuf</i>A and UPA markers to identify algae in metabarcoding surveys.  </b><br />
		<i>Data Brief</i> 11: 273-276 <br />
		DOI: <a href="https://dx.doi.org/10.1016/j.dib.2017.02.013">10.1016/j.dib.2017.02.013</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Maggs C.A., West J.A. & Verbruggen H. (2017)  <br />
		<b>Analysis of chloroplast genomes and a supermatrix inform reclassification of the Rhodomelaceae (Rhodophyta). </b> <br />
		<i>Journal of Phycology</i> 53: 920-937 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12553">10.1111/jpy.12553</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., Barbara I., Cremades J., Verbruggen H. & Maggs C.A. (2017)  <br />
		<b>Three new cryptogenic species in the tribe Polysiphonieae (Rhodomelaceae, Rhodophyta).  </b><br />
		<i>Phycologia</i> 56: 605-623 <br />
		DOI: <a href="https://doi.org/10.2216/17-17.1">10.2216/17-17.1</a>
	</td></tr>
	<tr><td>
		Verbruggen H., Marcelino V.R., Guiry M.D., Cremen M.C.M. & Jackson C. (2017)  <br />
		<b>Phylogenetic position of the coral symbiont <i>Ostreobium</i> (Ulvophyceae) inferred from chloroplast genome data.</b>  <br />
		<i>Journal of Phycology</i> 53: 790-803 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12540">10.1111/jpy.12540</a>
	</td></tr>
	<tr><td>
		Steen F., Aragay J., Zuljevic A., Verbruggen H., Mancuso F.P., Bunker F., Vitales D., Gomez Garreta A. & De Clerck O. (2017)  <br />
		<b>Tracing the introduction history of the brown seaweed <i>Dictyota cyanoloma</i> (Phaeophyceae, Dictyotales) in Europe. </b> <br />
		<i>European Journal of Phycology</i> 52: 31-42 <br />
		DOI: <a href="https://doi.org/10.1080/09670262.2016.1212998">10.1080/09670262.2016.1212998</a>
	</td></tr>
	<tr><td>
		Diaz-Tapia P., McIvor L., Freshwater D.W., Verbruggen H., Wynne M.J. & Maggs C.A (2017)  <br />
		<b>The genera <i>Melanothamnus</i> Bornet & Falkenberg and <i>Vertebrata</i> S.F. Gray constitute well-defined clades of the red algal tribe Polysiphonieae (Rhodomelaceae, Ceramiales). </b> <br />
		<i>European Journal of Phycology</i> 52: 1-30 <br />
		DOI: <a href="https://doi.org/10.1080/09670262.2016.1256436">10.1080/09670262.2016.1256436</a>
	</td></tr>
	<tr><td>
		Ximenes C.F., Cassano V., Oliveira-Carvalho M.F., Bandeira-Pedrosa M.E., Gurgel C.F.D., Verbruggen H. & Barreto-Pereira S.M. (2017) <br />
		<b>Systematics of the genus <i>Halimeda</i> (Bryopsidales, Chlorophyta) in Brazil including the description of <i>Halimeda jolyana</i> sp. nov.</b> <br />
		<i>Phycologia</i> 56: 369-381 <br />
		DOI: <a href="https://doi.org/10.2216/16-77.1">10.2216/16-77.1</a>
	</td></tr>
	<tr><td>
		Verbruggen H., Brookes M.J. & Costa J.F. (2017)  <br />
		<b>DNA barcodes and morphometric data indicate that <i>Codium fragile</i> may consist of two species (Bryopsidales, Chlorophyta).</b>  <br />
		<i>Phycologia</i> 56: 54-62 <br />
		DOI: <a href="https://doi.org/10.2216/16-54.1">10.2216/16-54.1</a>
	</td></tr>
</table>





<h2>2016</h2>
<table>
	<tr><td>
		Rindi F. & Verbruggen H. (2016)  <br />
		<b>Taxonomic reshuffling of the Cladophoraceae.</b>  <br />
		<i>Journal of Phycology</i> 52: 901-904 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12459">10.1111/jpy.12459</a>
	</td></tr>
	<tr><td>
		Costa J.F., Lin S.M., Macaya E.C., Fernandez-Garcia C. & Verbruggen H. (2016)  <br />
		<b>Chloroplast genomes as a tool to resolve red algal phylogenies: a case study in the Nemaliales.</b> <br />
		<i>BMC Evolutionary Biology</i> 16: 205 <br />
		DOI: <a href="https://doi.org/10.1186/s12862-016-0772-3">10.1186/s12862-016-0772-3</a>
	</td></tr>
	<tr><td>
		Alos J., Tomas F., Terrados J., Verbruggen H. & Ballesteros E. (2016) <br />
		<b>Fast-spreading green beds of recently introduced <i>Halimeda incrassata</i> (Bryopsidales, Chlorophyta) invade Mallorca Island (NW Mediterranean Sea).  </b><br />
		<i>Molecular Ecology Progress Series</i> 558: 153-158 <br />
		DOI: <a href="https://doi.org/10.3354/meps11869">10.3354/meps11869</a>
	</td></tr>
	<tr><td>
		Marcelino V.R., Cremen M.C.M., Jackson C.J., Larkum A. & Verbruggen H. (2016)  <br />
		<b>Evolutionary dynamics of chloroplast genomes in low light: a case study of the endolithic green alga <i>Ostreobium quekettii</i>.</b>  <br />
		<i>Genome Biology and Evolution</i> 8: 2939-2951 <br />
		DOI: <a href="https://doi.org/10.1093/gbe/evw206">10.1093/gbe/evw206</a> &mdash; preprint: <a href="https://doi.org/10.1101/049833">10.1101/049833</a>
	</td></tr>
	<tr><td>
		Marcelino V.R. & Verbruggen H. (2016)  <br />
		<b>Multi-marker metabarcoding of coral skeletons reveals a rich microbiome and diverse evolutionary origins of endolithic algae.</b>  <br />
		<i>Scientific Reports</i> 6: 31508 <br />
		DOI: <a href="https://dx.doi.org/10.1038/srep31508">10.1038/srep31508</a>
	</td></tr>
	<tr><td>
		Wetherbee R. & Verbruggen H. (2016)  <br />
		<b><i>Kraftionema allantoideum</i>, a new genus and family of Ulotrichales (Chlorophyta) adapted for survival in high intertidal pools.</b>  <br />
		<i>Journal of Phycology</i> 52: 704-715 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12447">10.1111/jpy.12447</a>
	</td></tr>
	<tr><td>
		West J.A., Zuccarello G.C., Loiseaux de Goer S., Stavrias L.A. & Verbruggen H. (2016)  <br />
		<b><i>Rhodenigma contortum</i>, an obscure new genus and species of Rhodogorgonales (Rhodophyta) from Western Australia. </b> <br />
		<i>Journal of Phycology</i> 52: 397-403 <br />
		DOI: <a href="https://doi.org/10.1111/jpy.12402">10.1111/jpy.12402</a>
	</td></tr>
	<tr><td>
		Cremen M.C.M., Huisman J.M., Marcelino V.R. & Verbruggen H. (2016)  <br />
		<b>Taxonomic revision of <i>Halimeda</i> in southwestern Australia.</b>  <br />
		<i>Australian Systematic Botany</i> 29: 41-54 <br />
		DOI: <a href="https://doi.org/10.1071/SB15043">10.1071/SB15043</a>
	</td></tr>
	<tr><td>
		Fernandez-Garcia C., Wysor B., Riosmena-Rodriguez R., Pena-Salamanca E. & Verbruggen H. (2016)  <br />
		<b>DNA-assisted identification of <i>Caulerpa</i> (Caulerpaceae, Chlorophyta) reduces species richness estimates for the Eastern Tropical Pacific. </b> <br />
		<i>Phytotaxa</i> 252: 185-204 <br />
		DOI: <a href="https://dx.doi.org/10.11646/phytotaxa.252.3.2">10.11646/phytotaxa.252.3.2</a>
	</td></tr>
	<tr><td>
		Lam D.W., Verbruggen H., Saunders G.W., Vis M.L. (2016)  <br />
		<b>Multigene phylogeny of the red algal subclass Nemaliophycidae.  </b><br />
		<i>Molecular Phylogenetics and Evolution</i> 94: 730-736 <br />
		DOI: <a href="https://doi.org/10.1016/j.ympev.2015.10.015">10.1016/j.ympev.2015.10.015</a>
	</td></tr>
</table>


<h2>Older work</h2>
<p>For older work, see <a href="https://scholar.google.com.au/citations?user=wrySKuIAAAAJ&hl=en">Google Scholar</a>.</p>

								</article>
							</div>

					</div>
				</div>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
