<!DOCTYPE HTML>
<!--
	Escape Velocity by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Verbruggen lab - Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<style>
			.myColoredBackground {
				background-color: #F4F4F2;
				padding: 2px 15px 20px 15px;
				margin: 0 0 30px 0;
			}
			.myWhiteBackground {
				background-color: #FFFFFF;
				padding: 2px 15px 20px 15px;
				margin: 0 0 30px 0;
			}
		</style>
	</head>
	<body class="left-sidebar is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<section id="header" class="wrapper">

					<!-- Logo -->
						<div id="logo">
							<h1><a href="index.html">Verbruggen laboratory</a></h1>
							<p>Algal evolution and Coral holobiont biology</p>
						</div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="index.html">Home</a></li>
								<li><a href="people.html">People</a></li>
								<li><a href="research.html">Research</a></li>
								<li><a href="publications.html">Publications</a></li>
								<li><a href="resources.html">Resources</a></li>
								<!--<li><a href="software.html">No Sidebar</a></li>-->
							</ul>
						</nav>

				</section>

			<!-- Main -->
				<section id="main" class="wrapper style2">
					<div class="title">Resources</div>
					<div class="container">
						<div class="row gtr-150">
							<div class="col-4 col-12-medium">

								<!-- Sidebar -->
									<div id="sidebar" class="myColoredBackground">
										<section class="box">
											<p style="font-size:1.6em; padding: 20px 0 0 0; margin: 0 0 15px 0; font-weight: bold;">Niche modeling</p>
											
											<a href="https://www.bio-oracle.org/index.php"><img src="resources/BioORACLE.png" width="100%"/></a>
											<p>Bio-ORACLE v.2.2 now out: a much-expanded version of our popular environmental data for marine niche modeling.</p>
											<a href="https://www.bio-oracle.org/index.php" class="button style3">To Bio-ORACLE!</a>
										</section>
										<section class="box">
											
											<ul class="style2">
												<li>
													<article class="box post-excerpt">
														<img class="image left" src="resources/MMS_thumbnail.jpg" alt="" />
														<h3>MMS &ndash; Maxent Model Surveyor</h3>
														<p>Evaluate predictor sets and model complexity for Maxent.<br />
														<a href="https://github.com/hverbruggen/MMS" class="button style32">Github</a></p>
													</article>
												</li>
												<li>
													<article class="box post-excerpt">
														<img class="image left" src="resources/OccurrenceThinner_thumbnail.jpg" alt="" />
														<h3>OccurrenceThinner</h3>
														<p>Distance-based thinning of species occurrence data.<br />
														<a href="https://github.com/hverbruggen/OccurrenceThinner" class="button style32">Github</a></p>
													</article>
												</li>
												<li>
													<article class="box post-excerpt">
														<img class="image left" src="resources/RasterTools_thumbnail.jpg" alt="" />
														<h3>RasterTools</h3>
														<p>Java tools for raster and vector GIS procedures common in niche modeling.<br />
														<a href="https://github.com/hverbruggen/RasterTools" class="button style32">Github</a></p>
													</article>
												</li>
											</ul>
										</section>

									</div>

							</div>
							<div class="col-8 col-12-medium imp-medium">

								<!-- Content -->
									<div id="content">

<!-- use this for 2-column content
										<div class="myColoredBackground">
											<article class="box post">
											<p style="font-size:1.6em; padding: 20px 0 0 0; margin: 0 0 15px 0; font-weight: bold;">Algae data resources</p>
											
												<div class="row gtr-150">
													<div class="col-6 col-12-small">
														<section class="box">
															<h2>The Ostreobium genome</h2>
															<a href="https://doi.org/10.5281/zenodo.4391443"><img src="resources/OstreobiumGenome.jpg" width="100%"/></a>
															<p>Annotations and additional data for the <i>Ostreobium</i> nuclear genome.<br />
															<a href="https://doi.org/10.5281/zenodo.4391443" class="button style31">Get data</a></p>
														</section>
													</div>
													<div class="col-6 col-12-small">
														<section class="box">
															<h2>Metabarcoding reference data</h2>
															<a href="https://doi.org/10.26188/c.5132798.v1"><img src="resources/MetabarcodingReferenceData.jpg" width="100%"/></a>
															<p>UPA and <i>tuf</i>A reference data for identification of algae in metabarcoding.<br />
															<a href="https://doi.org/10.26188/c.5132798.v1" class="button style31">Get data</a></p>
															
														</section>
													</div>
												</div>
											</article>
										</div>
-->

										<div class="myColoredBackground">
											<article class="box post">
											<p style="font-size:1.6em; padding: 20px 0 0 0; margin: 0 0 15px 0; font-weight: bold;">Algae data resources</p>

												<div class="row aln-center">
													<div class="col-4 col-12-medium">
														<section class="highlight">
															<h2>Ostreobium genome and strain resources</h2>
															<a href="resources/Ostreobium_Resources.html"><img src="resources/OstreobiumGenome.jpg" width="100%"/></a>
															<p>List of available <i>Ostreobium</i> strains and annotated data for the nuclear genome sequence.<br />
															<a href="resources/Ostreobium_Resources.html" class="button style31">Get data</a></p>
														</section>
													</div>
													<div class="col-4 col-12-medium">
														<section class="highlight">
															<h2>Pedinophyte YPF701 nuclear genome</h2>
															<a href="resources/Pedinophyte_YPF701.html"><img src="resources/Pedinophyte_YPF701.jpg" width="100%"/></a>
															<p>Annotations and additional data for the Pedinophyte YPF701 nuclear genome.<br />
															<a href="resources/Pedinophyte_YPF701.html" class="button style31">Get data</a></p>
														</section>
													</div>
													<div class="col-4 col-12-medium">
														<section class="highlight">
															<h2>Algal metabarcoding reference data</h2>
															<a href="https://doi.org/10.26188/c.5132798.v1"><img src="resources/MetabarcodingReferenceData.jpg" width="100%"/></a>
															<p>A set of UPA and <i>tuf</i>A reference sequences for identification of algae in metabarcoding surveys.<br />
															<a href="https://doi.org/10.26188/c.5132798.v1" class="button style31">Get data</a></p>
														</section>
													</div>
												</div>

											</article>
										</div>
												


									<div class="myColoredBackground">
										<article class="box post">
										<p style="font-size:1.6em; padding: 20px 0 0 0; margin: 0 0 15px 0; font-weight: bold;">Metagenomic data mining</p>
										
											<div class="row gtr-150">
												<div class="col-6 col-12-medium">
													<section class="box">
														<header>
															<h2>Metaphor</h2>
														</header>
														<p>Lean Snakemake workflow for metagenome assembly, annotation and binning.<br/>
														<a href="https://github.com/vinisalazar/metaphor" class="button style31">Github</a></p>
													</section>
												</div>
												<div class="col-6 col-12-medium">
													<section class="box">
														<header>
															<h2>CORGI</h2>
														</header>
														<p>Indentification of organelle genome contigs from metagenome assemblies.<br/>
														<a href="https://github.com/rbturnbull/corgi" class="button style31">Github</a></p>
													</section>
												</div>
											</div>
											<br />
											<div class="row gtr-150">
												<div class="col-6 col-12-medium">
													<section class="box">
														<header>
															<h2>MetaGenePipe</h2>
														</header>
														<p>Flexible and scalable metagenomics pipeline implemented in WDL.<br/>
														<a href="https://github.com/ParkvilleData/MetaGenePipe" class="button style31">Github</a></p>
													</section>
												</div>
												<div class="col-6 col-12-medium">
													<section class="box">
														<header>
															<h2>Melbourne Metagenomics Archive</h2>
														</header>
														<p>Browse metagenome data and eukaryotic species discovered therein.<br/>
														<a href="https://mma.unimelb.edu.au/" class="button style31">Browse</a></p>
													</section>
												</div>
											</div>
										</article>
									</div>



									<div class="myColoredBackground">
										<article class="box post">
											<p style="font-size:1.6em; padding: 20px 0 0 0; margin: 0 0 15px 0; font-weight: bold;">Tools for phylogenetics</p>
											
											<p><b>SiteStripper</b>, a Perl script to 'site strip' alignments, i.e. remove a given percentage of fast-evolving sites. <br />
											<a href="https://github.com/hverbruggen/SiteStripper" class="button style32">Github</a></p>

											<p><b>TreeGradients</b>, a phylogenetic tree drawing program using color or grayscale gradients to trace continuous characters along the tree. It allows for quite sophisticated color ramps. <br />
											<a href="https://github.com/hverbruggen/TreeGradients" class="button style32">Github</a></p>

											<p><b>gDAM</b> (graphical Data Availability Matrix), a script that draws a graphical representation of data availability in a concatenated sequence alignment. <br />
											<a href="https://github.com/hverbruggen/gDAM" class="button style32">Github</a></p>

											<p><b>Partitioned Model Tester</b>, a script that facilitates selecting partitioning strategies and models of sequence evolution for phylogenetics. We still provide the software, but be aware that support is discontinued and we recommend IQ-Tree or PartitionFinder for this. <br />
											<a href="https://github.com/hverbruggen/PMT" class="button style32">Github</a></p>
										</article>
									</div>



							</div>
						</div>
					</div>
				</section>

			<!-- Highlights -->
				<section id="highlights" class="wrapper style3">
					<div class="title">Resources from papers</div>
					<div class="container">
						<p>Original sequence data associated with our research papers gets submitted to ENA/NCBI/DDBJ, and we're not linking to that here. New software that we develop for our work is typically hosted on Github or similar repositories and linked above. The purpose of this list is to post additional resources when we consider they provide substantial added value for the research community, or whenever someone requests them from us. We realise that this list is incomplete, and if there are any resources you think we should add, shoot us an email and we will endeavour to do so.</p>
						
						<p>
							<a href="https://doi.org/10.26188/5e5468e745638" class="button style32">Get data</a>
							Rindi et al. (2020) Phylogeography of the Mediterranean green seaweed <i>Halimeda tuna</i>. <i>Journal of Phycology</i> 56: 1109-1113
						</p>

						<p>
							<a href="https://doi.org/10.26188/c.5048294.v1" class="button style32">Get data</a> 
							Bringloe et al. (2020) Phylogeny and evolution of the brown algae. <i>Critical Reviews in Plant Sciences</i> 39: 281-321
						</p>

						<p>
							<a href="https://doi.org/10.6084/m9.figshare.1290723.v3" class="button style32">Get data</a> 
							Verbruggen & Costa (2015) Molecular survey of <i>Codium</i> species diversity in southern Madagascar. <i>Cryptogamie Algologie</i> 36: 171-187
						</p>

						<p>
							<a href="https://figshare.com/projects/Laurencia_chloroplast_genome/89816" class="button style32">Get data</a> 
							Verbruggen & Costa (2015) The plastid genome of the red alga <i>Laurencia</i>. <i>Journal of Phycology</i> 51: 586-589
						</p>

						<p>
							<a href="https://doi.org/10.6084/m9.figshare.c.5134031.v1" class="button style32">Get data</a> 
							Verbruggen et al. (2013) Improving transferability of introduced speciesâ€™ distribution models: new tools to forecast the spread of a highly invasive seaweed. <i>PLoS One</i> 8: e68337
						</p>

						<p>
							<a href="resources/JPY-48-326.zip" class="button style32">Get data</a> 
							Ciancia et al. (2012) Characterization of cell wall polysaccharides of the coencocytic green seaweed <i>Bryopsis plumosa</i> (Bryopsidaceae, Chlorophyta) from the Argentine coast. <i>Journal of Phycology</i> 48: 326-335
						</p>

						<p>
							<a href="resources/EJP-47-355.zip" class="button style32">Get data</a> 
							Oliveira-Carvalho et al. (2012) Phylogenetic analysis of Codium species from Brazil, with the description of the new species <i>C. pernambucensis</i> (Bryopsidales, Chlorophyta). <i>European Journal of Phycology</i> 47: 355-365
						</p>

						<p>
							<a href="resources/EJP-47-216.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2012) The new species <i>Codium recurvatum</i> from Tanzania. <i>European Journal of Phycology</i>47: 216-222
						</p>

						<p>
							<a href="resources/JPY-48-1090.zip" class="button style32">Get data</a> 
							Verbruggen & Schils (2012) <i>Rhipilia coppejansii</i>, a new coral reef-associated species from Guam (Bryopsidales, Chlorophyta). <i>Journal of Phycology</i> 48: 1090-1098
						</p>

						<p>
							<a href="resources/PHY-50-403.zip" class="button style32">Get data</a> 
							Coppejans et al. (2011) <i>Rhipidosiphon lewmanomontiae</i> sp. nov. (Bryopsidales, Chlorophyta), a calcified udoteacean alga from the central Indo-Pacific based on morphological and molecular investigations. <i>Phycologia</i> 50: 403-412
						</p>

						<p>
							<a href="resources/BEB-10-16.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2010) Data mining approach identifies research priorities and data requirements for resolving the red algal tree of life. <i>BMC Evolutionary Biology</i> 10: 16
						</p>

						<p>
							<a href="resources/GEB-18-393.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2009) Macroecology meets macroevolution: evolutionary niche dynamics in the seaweed <i>Halimeda</i>. <i>Global Ecology and Biogeography</i> 18: 393-405
						</p>

						<p>
							<a href="resources/BEB-9-39.zip" class="button style32">Get data</a> 
							Cocquyt et al. (2009) Gain and loss of elongation factor genes in green algae. <i>BMC Evolutionary Biology</i> 9: 39
						</p>

						<p>
							<a href="resources/JPY-45-726.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2009) Phylogenetic analysis of <i>Pseudochlorodesmis</i> strains reveals cryptic diversity above the family level in the siphonous green algae (Bryopsidales, Chlorophyta). <i>Journal of Phycology</i> 45: 726-731
						</p>

						<p>
							<a href="resources/MPE-50-642.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2009) A multi-locus time-calibrated phylogeny of the siphonous green algae. <i>Molecular Phylogenetics and Evolution</i> 50: 642-653
						</p>

						<p>
							<a href="resources/PHY-46-513.zip" class="button style32">Get data</a> 
							Verbruggen et al. (2007) <i>Halimeda pygmaea</i> and <i>Halimeda pumila</i> (Bryopsidales, Chlorophyta): two new dwarf species from fore reef slopes in Fiji and the Bahamas. <i>Phycologia</i> 46: 513-520
						</p>




					</div>
				</section>


		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>